<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="generalFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="languageLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="languageComboBox">
         <item>
          <property name="text">
           <string>日本語</string>
          </property>
          <property name="data">
           <string>ja_JP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
          <property name="data">
           <string>en</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="generalVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="windowTab">
      <attribute name="title">
       <string>Window</string>
      </attribute>
      <layout class="QVBoxLayout" name="windowVerticalLayout">
       <item>
        <widget class="QGroupBox" name="windowBehaviorGroupBox">
         <property name="title">
          <string>Window Behavior</string>
         </property>
         <layout class="QVBoxLayout" name="windowBehaviorLayout">
          <item>
           <widget class="QCheckBox" name="alwaysOnTopCheckBox">
            <property name="text">
             <string>Always on top</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rememberWindowSizeCheckBox">
            <property name="text">
             <string>Remember window size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rememberWindowPositionCheckBox">
            <property name="text">
             <string>Remember window position</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="windowVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>